txmongo/database.py

```diff
@@ -44,14 +45,17 @@ def connection(self):
     def write_concern(self):
         return self.__write_concern or self.__factory.write_concern
 
+    @timeout
     @defer.inlineCallbacks
     def command(self, command, value=1, check=True, allowable_errors=None, **kwargs):
         if isinstance(command, (bytes, unicode)):
             command = SON([(command, value)])
-        command.update(kwargs)
+        options = kwargs
+        options.pop("_deadline", None)
+        command.update(options)
 
         ns = self["$cmd"]
-        response = yield ns.find_one(command)
+        response = yield ns.find_one(command, **kwargs)
```

Comment:
`_deadline` is not passed to `find_one`, because you popped it from `kwargs`. Did you meant this?